server/notebooks/views.py

```diff
@@ -33,13 +33,18 @@ def notebook_view(request, pk):
         revision = get_object_or_404(NotebookRevision, pk=int(request.GET["revision"]))
     else:
         revision = notebook.revisions.first()
+    files = [
+        {"filename": file.filename, "id": file.id, "lastUpdated": file.last_updated.isoformat()}
+        for file in File.objects.filter(notebook_id=pk).order_by("-last_updated")
+    ]
```

Comment:
man i miss list comprehensions...

Reply:
Ideally the file model would be ordered by last_updated, which would make the last clause unnecessary.

We already do this for e.g. NotebookRevision:

https://github.com/iodide-project/iodide/blob/f7699e90da6ad52e4f36c44d765afdae70c7050c/server/notebooks/models.py#L59

(this can be done in a followup, if you prefer)